getting-started/openstack/installation/ubuntu.md

```diff
@@ -61,7 +48,7 @@ These steps are detailed in this section.
 
     ```
     apt-get install -y python3-pip
-    pip3 install git+https://github.com/dims/etcd3-gateway.git@5a3157a122368c2314c7a961f61722e47355f981
+    pip3 install git+https://opendev.org/openstack/etcd3gw.git@dc9f55975ecd5b291eb1014b9fe8393dcd6d7e5c
```

Comment:
Is there a strong reason to make this change?  The github.com/dims link still works, and our own internal testing uses it, so I'd prefer to keep it unless these is a strong reason to change.